judo-runtime-core-jsl-itest/models/pom.xml

```diff
@@ -43,7 +40,7 @@
         <module>TransferAggregationModel</module>
         <module>TransferAssociationModel</module>
         <module>TransferConstructorModel</module>
-        <module>RecursiveCompositionModel</module>
+<!--        <module>RecursiveCompositionModel</module> -->
```

Comment:
Why is it removed?

Reply:
Added back without automapped test

Reply:
[75268a2](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/241/commits/75268a2d52d7d67a4551a6495568f3a0aae606f6)

Reply:
[ebb9ce9](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/241/commits/ebb9ce9bb6dfd790c87e92d80b76f843e4a2a954)